intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -2401,6 +2401,33 @@ def convert_date(value):
     }
 }
 
+# https://www.shadowserver.org/what-we-do/network-reporting/sinkhole-dns-report/
+accessible_sinkhole_dns = {
+    'required_fields': [
+        ('time.source', 'timestamp', add_UTC_to_timestamp),
+        ('source.ip', 'ip', validate_ip),
+        ('source.port', 'port', convert_int)
+    ],
+    'optional_fields': [
+        ('source.reverse_dns', 'host'),
+        ('source.asn', 'asn', convert_int),
+        ('source.geolocation.cc', 'geo'),
+        ('source.geolocation.region', 'region'),
+        ('extra.', 'type'),
+        ('extra.', 'count', convert_int),
+        ('extra.', 'query'),
+        ('extra.', 'response'),
+        ('extra.', 'tag'),
+        ('extra.', 'sector'),
+        ('extra.', 'naics', convert_int),
+    ],
+    'constant_fields': {
+        'classification.identifier': 'accessible-sinkholedns',
+        'classification.taxonomy': 'vulnerable',
+        'classification.type': 'vulnerable service',
```

Comment:
I don't think that the classifications fits.

[Shadowserver's description is](https://www.shadowserver.org/what-we-do/network-reporting/sinkhole-dns-report/):
> This report lists DNS queries seen from recursive DNS servers for sinkholed domains. > Please note that the IP listed are not the same as the actual source IP of the client that is making the query and hence are likely not infected hosts. This report therefore is to be used primarily to support investigations into a threat, and not as a source of direct identification of infected hosts.

So the taxonomy is malicious code or other, not vulnerable. The event itself is neither a infected system, nor a C&C or related to malware distribution.

As the IoC is further not directly linked to a malware incident, I tending towards *other*. We could as well do a short "survey" on the RSIT mailinglist about what others think about this.

cc @th-certbund

Reply:
In the n6 parser we already use other/other/dns-query